promptsource/templates/lama/squad/templates.yaml

```diff
@@ -0,0 +1,62 @@
+dataset: lama
+subset: squad
+templates:
+  5df3ac5b-2968-4f64-9427-4ee2705ad4d9: !Template
+    answer_choices: null
+    id: 5df3ac5b-2968-4f64-9427-4ee2705ad4d9
+    jinja: "{{masked_sentence}} \nWrite a single word to replace the [MASK]. ||| {{obj_label}}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      original_task: true
+    name: single word
+    reference: ''
+  9068e083-4037-475a-a17f-1370d1d965cf: !Template
+    answer_choices: null
+    id: 9068e083-4037-475a-a17f-1370d1d965cf
+    jinja: 'Give a single word to fill the [MASK] that consistent with the following
+      sentence: {{masked_sentence}} ||| {{obj_label}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      original_task: true
+    name: consistent
+    reference: ''
+  9c4bc2f5-2eb6-4ccb-8c7d-5b387c066e31: !Template
+    answer_choices: null
+    id: 9c4bc2f5-2eb6-4ccb-8c7d-5b387c066e31
+    jinja: 'What is a correct word to replace [MASK] given the following context:
+      {{masked_sentence}} ||| {{obj_label}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      original_task: true
+    name: given_context
+    reference: ''
+  a27a9d7e-ddad-4944-a0f8-df5f26b55316: !Template
+    answer_choices: null
+    id: a27a9d7e-ddad-4944-a0f8-df5f26b55316
+    jinja: 'Given the following masked sentence: {{masked_sentence}} What is the correct
+      answer to fill the mask ? ||| {{obj_label}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      original_task: true
+    name: what_is
+    reference: ''
+  af58d7b9-6e15-4e82-b5d4-5e51cbac7196: !Template
+    answer_choices: null
+    id: af58d7b9-6e15-4e82-b5d4-5e51cbac7196
+    jinja: 'Predict a right word to fill at [MASK] in the following sentence: {{masked_sentence}}
```

Comment:
In a normal human context, we would not ask someone to "predict a right word".  We could maybe use "Suggest" or "Please provide"?